### DISCUSSION TITLE
希望增加可以把漫画页面放大的功能

### DISCUSSION THREAD

**Discussion by FightingBuiui** [xiaoyaocz/flutter_dmzj#114] [General]
手机看漫画有些漫画字太小了，希望可以双击屏幕放大或者双指触摸放大缩小

**Comment by oubeichen** [comment]
试了下应该是可以双指缩放的，就是特别容易和翻页冲突，估计手势检测有点问题。或者干脆弄成双击放大或者加减放大比较简单。

**Reply by FightingBuiui** [reply]
我刚试了下，左右翻页确实可以双指缩放，向下翻页的不行，我看漫画用习惯向下翻页了

**Comment by CatfishWen** [comment]
希望能够恢复之前版本的双击放大功能，单手操作时双击放大比较符合操作习惯👍
